src/test/java/use_case/cat/CatInteractorTest.java

```diff
@@ -103,7 +103,7 @@ void successIncreaseCatHappinessTest() {
         cat.setCatObjectCreated(true);
 
         final ChangeCatHappinessInputData inputData = new ChangeCatHappinessInputData(cat.getName(),
-                cat.getOwnerUsername(), cat, true, 20);
+                cat.getOwnerUsername(), true, 20);
```

Comment:
Getting Checkstyle errors for this. IntelliJ sets scope to not include tests, but change in IntelliJ Checkstyle settings. Set scope to include tests.

Reply:
Refactor: make separate unit tests for different use cases. Package structure in tests folder should match the main folder. Otherwise, we can't check for 100% coverage, because it only tests the CatDataAccessInterface